src/nft_contract/src/test/burn_private.nr

```diff
@@ -3,117 +3,120 @@ use crate::test::utils;
 use authwit::cheatcodes as authwit_cheatcodes;
 
 #[test]
-unconstrained fn nft_burn_private_self_success() {
-    // Setup and mint NFT to owner in private state
-    let token_id = 10000;
-    let (env, nft_contract_address, owner, _, _) =
-        utils::setup_and_mint_to_private(false, token_id);
+unconstrained fn test_burn_private() {
+    // Setup without account contracts first
+    let (env, nft_contract_address, owner, minter, recipient) = utils::setup_with_minter(false);
```

Comment:
why would we do 2 setups? it's preferrable to do 1 with account contracts, than once w/o and once with